src/screens/help.ts

```diff
@@ -132,12 +132,18 @@ export default async function helpScreen(): Promise<string> {
 			}
 
 			if (details.accepts) {
-				fullDescription += chalk.gray.italic(` (accepts: ${details.accepts.join(', ')})`);
+				const accepts =
+					typeof details.accepts !== 'function'
+						? details.accepts
+						: details.accepts.constructor.name === 'AsyncFunction'
+						? await details.accepts()
+						: (details.accepts() as string[] | number[]);
```

Comment:
Why does this type need to be coerced using `as string[] | number[]`?

Reply:
Would that cause TypeScript to better understand what's going on?

Reply:
TypeScript seems 'happy' if I *always* assume the callback function is `async` and `await` the response.

However, I don't really want to do that, as, it actually causes the synchronous case to be wrapped in a Promise and for the call to then be processed asynchronously (causing additional delay).

I *want* to use knowledge as to what type of callback function it is to know whether it really needs to be `await`ed or not.

Reply:
Hmmmm.  I did see some comments about some sneaky ways to detect the response type without actually calling the function.  hmmmm.

Reply:
What about my suggestion regarding using `instanceof`?

Reply:
We're going to have to call the function either way; all we need to do is decide whether the return value is a promise, and if so, `await` it.

Reply:
Yeah, so, the best I've been able to do so far (using `instanceof`) *still* needs to tell TypeScript that the call to the function is only going to be handing back an array versus potentially also being a `Promise<string[] | number[]>`.

Reply:
I *suspect* that `() => Promise<string[] | number[]> | () => string[] | number[]` is really being seen as just: `() => Promise<string[]|number[]> | string[] | number[]` and possibly because of that, it's hard for TypeScript to figure out how to separate them.  The 'await' case seems to be understood, but not just calling it directly.